src/main.rs

```diff
@@ -78,58 +80,54 @@ static RPC_CLIENT: Lazy<near_fetch::Client> = Lazy::new(|| {
+static RELAYER_ACCOUNT_ID: Lazy<String> =
+    Lazy::new(|| LOCAL_CONF.get("relayer_account_id").unwrap());
+static SHARED_STORAGE_ACCOUNT_ID: Lazy<String> =
+    Lazy::new(|| LOCAL_CONF.get("shared_storage_account_id").unwrap());
 static SIGNER: Lazy<KeyRotatingSigner> = Lazy::new(|| {
-    let paths = LOCAL_CONF.get::<Vec<String>>("keys_filenames")
+    let paths = LOCAL_CONF
+        .get::<Vec<String>>("keys_filenames")
         .expect("Failed to read 'keys_filenames' from config");
     KeyRotatingSigner::from_signers(paths.iter().map(|path| {
         InMemorySigner::from_file(Path::new(path))
             .unwrap_or_else(|err| panic!("failed to read signing keys from {path}: {err:?}"))
     }))
 });
-static SHARED_STORAGE_KEYS_FILENAME: Lazy<String> = Lazy::new(|| {
-    LOCAL_CONF.get("shared_storage_keys_filename").unwrap()
-});
-static WHITELISTED_CONTRACTS: Lazy<Vec<String>> = Lazy::new(|| {
-    LOCAL_CONF.get("whitelisted_contracts").unwrap()
-});
+static SHARED_STORAGE_KEYS_FILENAME: Lazy<String> =
+    Lazy::new(|| LOCAL_CONF.get("shared_storage_keys_filename").unwrap());
+static WHITELISTED_CONTRACTS: Lazy<Vec<String>> =
+    Lazy::new(|| LOCAL_CONF.get("whitelisted_contracts").unwrap());
 static USE_WHITELISTED_DELEGATE_ACTION_RECEIVER_IDS: Lazy<bool> = Lazy::new(|| {
-    LOCAL_CONF.get("use_whitelisted_delegate_action_receiver_ids").unwrap_or(false)
+    LOCAL_CONF
+        .get("use_whitelisted_delegate_action_receiver_ids")
+        .unwrap_or(false)
 });
 static WHITELISTED_DELEGATE_ACTION_RECEIVER_IDS: Lazy<Vec<String>> = Lazy::new(|| {
-    LOCAL_CONF.get("whitelisted_delegate_action_receiver_ids").unwrap()
-});
-static USE_REDIS: Lazy<bool> = Lazy::new(|| {
-    LOCAL_CONF.get("use_redis").unwrap_or(false)
+    LOCAL_CONF
+        .get("whitelisted_delegate_action_receiver_ids")
+        .unwrap()
 });
+static USE_REDIS: Lazy<bool> = Lazy::new(|| LOCAL_CONF.get("use_redis").unwrap_or(false));
 static REDIS_POOL: Lazy<Pool<RedisConnectionManager>> = Lazy::new(|| {
     let redis_cnxn_url: String = LOCAL_CONF.get("redis_url").unwrap();
     let manager = RedisConnectionManager::new(redis_cnxn_url).unwrap();
     Pool::builder().build(manager).unwrap()
 });
-static USE_FASTAUTH_FEATURES: Lazy<bool> = Lazy::new(|| {
-    LOCAL_CONF.get("use_fastauth_features").unwrap_or(false)
-});
-static USE_PAY_WITH_FT: Lazy<bool> = Lazy::new(|| {
-   LOCAL_CONF.get("use_pay_with_ft").unwrap_or(false)
-});
-static BURN_ADDRESS: Lazy<String> = Lazy::new(||{
-   LOCAL_CONF.get("burn_address").unwrap()
-});
-static USE_SHARED_STORAGE: Lazy<bool> = Lazy::new(|| {
-    LOCAL_CONF.get("use_shared_storage").unwrap_or(false)
-});
+static USE_FASTAUTH_FEATURES: Lazy<bool> =
+    Lazy::new(|| LOCAL_CONF.get("use_fastauth_features").unwrap_or(false));
+static USE_PAY_WITH_FT: Lazy<bool> =
+    Lazy::new(|| LOCAL_CONF.get("use_pay_with_ft").unwrap_or(false));
+static BURN_ADDRESS: Lazy<String> = Lazy::new(|| LOCAL_CONF.get("burn_address").unwrap());
```

Comment:
The burn address should be loaded in as an `AccountId` type just so we don't accidentally enter in an invalid value and forget to check much later into the code